### PR TITLE
Created set_parameters function in ps3eyedriver to expose set_<parm>

### PR DESCRIPTION
**Pull request by cboulay**
The psmoveapi does not send parameters that are appropriate for the PS3EYE, so some of the set functions are still commented out.

### CODE REVIEW

**Comment by thp** [comment]
These functions still need to be called by the PS Move API core library, right?

**Comment by cboulay** [comment]
Yes, that's correct. You can see how I call it [here](https://github.com/cboulay/psmoveapi/commit/9fa08177ae5e648f7616ca513b5354b8467b4fef#diff-5fa37dfa98cf5914ab1f52c74efe9eefR184). Is it possible to call functions declared in ps3eye.h directly from the psmoveapi? I was under the impression only ps3eyedriver.h -declared functions were available. If it is possible to call ps3eye.h functions (with just a simple #include) then that would probably be preferred. I attempted and failed to do so, but I may have just had the wrong syntax.

**Comment by cboulay** [comment]
I just tried again and was reminded why I chose to use ps3eyedriver instead of using the already available functions in ps3eye.h. I get the following whenever I try to include ps3eye.h: 
```
 'cstdio' file not found
#include <cstdio>
```
ps3eye.h already has include guards on it, so that's not it. I couldn't figure it out.

File: src/ps3eyedriver.cpp [hunk 1]
```diff
@@ -214,3 +214,33 @@ ps3eye_close(ps3eye_t *eye)
...
+    //eye->eye->setSharpness(sharpness);
+    //eye->eye->setContrast(contrast);
+    //eye->eye->setBrightness(brightness);
+    //eye->eye->setHue(hue);
+    //eye->eye->setRedBalance(redblc);
+    //eye->eye->setBlueBalance(blueblc);
+    //eye->eye->setFlip(flip_h, flip_v);
+    //ps3eye::PS3EYECam::updateDevices();
+    
+    return err;
```
Comment (thp) [inline src/ps3eyedriver.cpp]: `err` is never set to anything other than 0, so can remove `int err = 0;` from above and just have `return 0;` here

File: src/ps3eyedriver.h [hunk 2]
```diff
@@ -81,6 +84,15 @@ ps3eye_grab_frame(ps3eye_t *eye, int *stride);
  **/
 void
 ps3eye_close(ps3eye_t *eye);
+    
+int
+ps3eye_set_parameters(ps3eye_t *eye, bool autogain, bool awb, uint8_t gain, uint8_t exposure,
+                        uint8_t contrast, uint8_t brightness);
+/**
```
Comment (thp) [inline src/ps3eyedriver.h]: Comment should be above function header declaration. Mention what the result value means (0 = success, -1 = error).
As you document the possible ranges here, would it make sense to check (inside the functions) if the values are inside the range, and return an error if not?

**Comment by thp** [comment]
Added some comments, please have a look.

**Comment by cboulay** [comment]
I agree with your comments fully, but do you think that instead of something like ps3eye_set_parameters that the parameter-specific functions should be exposed? And maybe this can be done to mimic the CL Eye driver API as much as possible?

These functions are already in ps3eye.h/c but I cannot include ps3eye.h without running into a compiler error. Do you know what I'm doing wrong or is that behaving as intended?

**Pull request merged**